### DISCUSSION TITLE
React hooks

### DISCUSSION THREAD

**Discussion by genox** [swellstores/community#192] [Help] [answered]
Hi,

You mention a ContextProvider as well as hooks in the changelog. I assume this refers to the @swell/js and @swell/react packages that are available on npm.

I think it would be helpful to have these packages mentioned in your documentation. I wonder though, what featureset do the hooks cover? I assume it's mostly read ops, interaction with authentication, carts, checkout is probably to be done using @swell/js?

Does this mean you are retiring swell-js?

Which one is the preferred way to do things?

**Comment by maarten2424** [comment, marked as answer]
Hey @genox thanks for reaching out to us!
In our changelog we are referring to an internal project which currently is on hold.
I would suggest you have a look [at the documentation](https://developers.swell.is/storefronts/frameworks), there are two starter kit's that are using hooks from swell for react. 

`swell-js` won't be retired, and we'll continue to support, update and improve the library!
